virtManager/delete.py

```diff
@@ -508,10 +508,14 @@ def _get_title_text(self, devs):
             text = title
         return [title, text]
 
-    def _vm_active_status(self)
+    def _vm_active_status(self):
```

Comment:
whoops, this bit needs to be squashed into the previous patch. Each individual commit should be working and pass pylint. `git rebase --interactive` is your friend :)